pkg/cloud/services/eks/tags.go

```diff
@@ -68,11 +71,103 @@ func getTagUpdates(currentTags map[string]string, tags map[string]string) (untag
 	return untagKeys, newTags
 }
 
+func getASGTagUpdates(clusterName string, currentTags map[string]string, tags map[string]string) (tagsToDelete map[string]string, tagsToAdd map[string]string) {
+	officialASGTagsByEKS := []string{
+		"eks:cluster-name",
```

Comment:
Would you like to store these tags in a separate variable and then directly use them here?